latex/headers/stream.h

```diff
@@ -47,12 +51,16 @@ __width_manipulator__ setw(int width);
 class stream {
  public:
 
-  stream(size_t bufferSize, size_t maxStatementSize, handler& cgh);
+  stream(size_t totalBufferSize, size_t workItemBufferSize, handler& cgh);
 
   /* -- common interface members -- */
 
   size_t get_size() const;
 
+  size_t get_work_item_buffer_size() const;
```

Comment:
No.  Intent is to rename `get_max_statement_size()` to `get_work_item_buffer_size()` which is aligned with the constructor terminology.  It still returns the value passed as the first constructor arg.

Reply:
Perhaps that counts as deprecating, but in a weaker way than removal of a feature.

Reply:
I think this makes sense, if by renaming it we at some point remove `get_max_statement_size` then I would consider this to be a deprecation, simply because it is available in SYCL 1.2.1 but would not be available in a later revision.

To clarify I am fine with this, but if we are deprecating an interface I think we should mark it as deprecated in the specification so our intentions are clear.

Reply:
I think I clarified appropriately with the latest commit.